starboard/linux/x64x11/starboard_loader.cc

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dlfcn.h>
+#include <stdio.h>
+#include <time.h>
+
+#include "starboard/common/log.h"
+#include "starboard/configuration.h"
+#include "starboard/event.h"
+
+#define CONCATENATE_STRINGS(a, b) a b
+
+int main(int argc, char** argv) {
+  static const char* s_libraryPath1 =
+      "/usr/local/google/home/andrewsavage/cobalt/cobalt_src/out/"
```

Comment:
@y4vor @niranjanyardi what do we want to do about the loader here to make it general?

Reply:
Let's try to use relative path.

Reply:
Yes, but do we want to rely on a macro from GN or some other method of getting that